Jenkinsfile

```diff
@@ -61,6 +62,20 @@ pipeline {
             }
         }
 
+        stage('Linux test') {
```

Comment:
MJ - there was some issue with stage names containing spaces. We were removing them recently.